index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="kelwilson" />
+    <title>kelwilson's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="portfolio-wrap">
+      <div class="background-icons">
+        <div class="background- img">
+          <img
+            src="images/image_geometry_1.png"
+            class="background-image"
+            alt=""
+          />
+        </div>
+
+        <div class="toggle">
+          <div class="navigation">
+            <img src="images/ic_menu@2x.png" class="menu-btn" alt="" />
+          </div>
+        </div>
+        </div>
+    </div>
+    <div class="main-header">
+      <header class="portfolio-header">
+        <div class="header-div">
+          <h1 class="header">
+            Kelly <br />
+            Besong
+          </h1>
+          <p>
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Take a look at my works. If you like what you
+            see and have a project you need coded, don’t hesitate and contact
+            me.
+          </p>
+        </div>
+
+          <div class="btn">
+            <h2 class="start-btn">Start collaboration</h2>
+          </div>
+        </header>
```

Comment:
- [ ] This block actually serves the purpose of being a `section`, a block of specific information about something. Kindly switch this block wrap from a `header` tag to a `section` tag.